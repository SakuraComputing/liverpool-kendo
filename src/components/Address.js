import React from 'react';
import SimpleMap from './GoogleMap';

export const Address = () => {
    return (
        <div className="content-container">
            <p></p>
            <li>The Egg Dance Studio</li>
            <li>IM Marsh Campus</li>
            <li>Mossley Hill Road</li>
            <li>Aigburth</li>
            <li>Garston</li>
            <li>L17 6BD</li>

            <li>Wednesdays 7.30pm to 9.30pm</li>

            <li>Contact: 07752 969106</li>

            <div style={{width: '100%', height: '400px'}}>
                <SimpleMap/>
            </div>


        </div>

    );
};
export default Address;
