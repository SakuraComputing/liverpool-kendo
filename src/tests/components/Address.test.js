import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Address from '../../components/Address';

test('should render the Address page',() => {
    const wrapper = shallow(<Address />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});
