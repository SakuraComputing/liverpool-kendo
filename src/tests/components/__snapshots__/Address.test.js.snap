// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Address page 1`] = `
<div
  className="content-container"
>
  <h3>
     Where To Find Us
  </h3>
  <div>
    Childwall Sports and Science Academy
  </div>
  <div>
    Queens Drive
  </div>
  <div>
    Childwall
  </div>
  <div>
    L15 6XZ
  </div>
  <h3>
    Practice Session Time
  </h3>
  <li>
    Saturdays 2.00pm to 4.00pm
  </li>
  <li>
    Contact: 07515 546732
  </li>
  <p />
  <div
    style={
      Object {
        "height": "400px",
        "width": "100%",
      }
    }
  >
    <SimpleMap
      center={
        Object {
          "lat": 53.399373,
          "lng": 2.898834,
        }
      }
      zoom={16}
    />
  </div>
  <p>
    * Beginners always welcome but we ask you join the British Kendo Association after an introductory period.
  </p>
</div>
`;
